go.mod

```diff
@@ -1,11 +1,23 @@
 module github.com/patrickmcnamara/eiv
 
-go 1.15
+go 1.21.1
 
 require (
 	github.com/nfnt/resize v0.0.0-20180221191011-83c6a9932646
-	golang.org/x/exp v0.0.0-20201229011636-eab1b5eb1a03
-	golang.org/x/image v0.0.0-20201208152932-35266b937fa6
-	golang.org/x/mobile v0.0.0-20201217150744-e6ae53a27f4f
-	golang.org/x/sys v0.0.0-20200625212154-ddb9806d33ae // indirect
+	golang.org/x/exp/shiny v0.0.0-20231006140011-7918f672742d
+	golang.org/x/image v0.13.0
+	golang.org/x/mobile v0.0.0-20231006135142-2b44d11868fe
+)
+
+require (
+	9fans.net/go v0.0.4 // indirect
+	dmitri.shuralyov.com/gpu/mtl v0.0.0-20221208032759-85de2813cf6b // indirect
+	github.com/Plan9-Archive/libauth v0.0.0-20180917063427-d1ca9e94969d // indirect
+	github.com/emersion/go-sasl v0.0.0-20220912192320-0145f2c60ead // indirect
+	github.com/fhs/mux9p v0.3.1 // indirect
+	github.com/go-gl/glfw/v3.3/glfw v0.0.0-20221017161538-93cebf72946b // indirect
+	github.com/jezek/xgb v1.1.0 // indirect
+	github.com/knusbaum/go9p v1.18.0 // indirect
+	golang.org/x/sys v0.13.0 // indirect
+	supertxt.net/git/shiny-wire v0.0.5 // indirect
```

Comment:
I see now that there's the shiny-wire dependency in here. This should probably be removed before merging.